ludwig/utils/image_utils.py

```diff
@@ -30,16 +31,15 @@
 logger = logging.getLogger(__name__)
 
 
+@functools.lru_cache(maxsize=32)
```

Comment:
Is the purpose this cache to avoid downloading the image twice when we infer the image dimensions (1) and then process it (2)?

Reply:
Yeah, or to avoid downloading images with the same url in each step